docs/docs/en/guide/resource/configuration.md

```diff
@@ -2,13 +2,23 @@
 
 The Resource Center is usually used for operations such as uploading files, UDF functions, and task group management. You can appoint the local file directory as the upload directory for a single machine (this operation does not need to deploy Hadoop). Or you can also upload to a Hadoop or MinIO cluster, at this time, you need to have Hadoop (2.6+) or MinIO or other related environments.
 
+## Local File Resource Configuration
+For a single machine, you can choose to use local file directory as the upload directory (no need to deploy Hadoop) by making the following configuration.
+
+### Configuring the common.properties
+
+Configure the file in the following paths: `api-server/conf/common.properties` and `worker-server/conf/common.properties`.
+
+- Change 'data.basedir.path' to the local directory path. Please make sure the directory exists and the user who deploy dolphinscheduler have read and write permissions, such as: `data.basedir.path=/tmp/dolphinscheduler`.
```

Comment:
Same as the previous comment
```suggestion
- Change `data.basedir.path` to the local directory path. Please make sure the directory exists and the user who deploy dolphinscheduler have read and write permissions, such as: `data.basedir.path=/tmp/dolphinscheduler`.
```

Reply:
I think it will auto-create directory if not exists, so maybe we could delete the `Please make sure the directory exists` https://github.com/apache/dolphinscheduler/blob/996790ce9ea48bd96aed6687d31afe931fc70893/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HadoopUtils.java#L103-L112

Reply:
cc @GavinGYM